sdk/inc/azure/core/az_credentials.h

```diff
@@ -135,17 +123,18 @@ AZ_NODISCARD az_result az_credential_client_secret_init(
  * @param tenant_id An Azure tenant ID.
  * @param client_id An Azure client ID.
  * @param client_secret An Azure client secret.
- * @param authority Authentication authority URL to set.
+ * @param authority Authentication authority URL to set. Passing AZ_SPAN_NULL initializes credential
+ * with default authority (Azure AD global authority - "https://login.microsoftonline.com/")
  *
- * @note #authority example: "https://login.microsoftonline.us/".
+ * @note non-NULL #authority example: "https://login.microsoftonline.us/".
```

Comment:
authority isn't a type, function or macro. It's a parameter name.

```suggestion
 * @note Example of a non-NULL \p authority string: "https://login.microsoftonline.us/".
```